app/frontend/Calendar/EditEvent/TimeBox.svelte

```diff
@@ -94,7 +95,7 @@
 
   $: showTimezone = $event.timezone != myTimezone();
   $: $event.endTime, checkEndTime();
-  $: isMultipleDays = $event.startTime && $event.endTime && $event.endTime.toLocaleDateString() != $event.startTime.toLocaleDateString();
+  $: isMultipleDays = $event.startTime && $event.endTime && new Date($event.endTime.getTime() - event.allDay).toDateString() != $event.startTime.toDateString();
```

Comment:
`event.allDay`  is a boolean. Substracting a boolean from a number is not a valid operation.
I also don't understand why the old code was wrong and what this new code is trying to do. Can you rewrite this line, please?